frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -209,6 +208,9 @@ const PersonForm = (props: Props) => {
             options={ROLE_VALUES}
             defaultSelect={true}
           />
+          {patient.role === "STUDENT" && (
```

Comment:
Concern only tangentially related to this PR: in `constants.d.ts`, the `Role` type is a union of several string constants. There also exists, in `constants/index.tsx`, `ROLE_VALUES`, which enumerates some label/value pairs for this same type.

I wonder if the `Role` type should be, instead, a string-valued Enum which then keys the VALUES enum? 

But this refactor is beyond the scope of this magic string constant, which I hope, would be deprecated if/when these display values become configurable in the future